CONTRIBUTING.md

```diff
@@ -0,0 +1,117 @@
+  --to-release-image quay.io/vrutkovs/okd-release:4.4
+```
+This command would copy images, referenced in `registry.svc.ci.openshift.org/origin/release:4.4.0-0.okd-2020-03-13-053843` image to `quay.io/vrutkovs/okd-content` and create a new release image `quay.io/vrutkovs/okd-release:4.4`. The new release image would use `quay.io/vrutkovs/okd-content` as an 
+additional source of images.
+
+`release mirror` command would also print out the mirroring configuration for the installer.
+
+See https://docs.okd.io/latest/installing/installing_bare_metal/installing-restricted-networks-bare-metal.html
+
+# Making changes
+
+`oc adm release` can be used to create new releases, amending existing releases. If one of the images 
+needs to be updated use `release new` subcommand.
+
+For example, lets create a new release with updated Prometheus image. Use `release info` command 
+to print the pullspec used in the existing image:
+```
+$ oc adm release info registry.svc.ci.openshift.org/origin/release:4.4 --pullspecs | grep prometheus
+  k8s-prometheus-adapter                         registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:12bac47c71cb7ef36b6ee7b78e0476fbfb8a67bbf61ac42c461c17c98ac850a6
+  prometheus                                     registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:5af0373659974782379d90d9a174352dd8f85cb7327cc48ef36cae4e8ba5903f
+  prometheus-alertmanager                        registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:25bed531ccb0ff16ce19b927265f03cb9b2d572caa224ef302002269e925d83c
+  prometheus-config-reloader                     registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:deacbd618b3c037cc8c99a83db2c2a1053db517b0a0bfdfdeb309591559c3eea
+  prometheus-node-exporter                       registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:c199e7353642ed1a4237416055a75b0e415034c7ec48bbc8ae8d12b72552f819
+  prometheus-operator                            registry.svc.ci.openshift.org/origin/4.4-2020-04-14-172428@sha256:ec28b9dc5ad9184d0d70b85e5bc618c809084b293cbc57c215bf845bf7147b2b
+```
+
+In `release new` command a list of replaced images can be specified:
+```
+$ oc adm -a /path/to/pull_secret.json \
+  release new \
+  --from-release registry.svc.ci.openshift.org/origin/release:4.4 \
+  --to-image quay.io/vrutkovs/okd-release:4.4-updated-prometheus \
+  prometheus=docker.io/prom/prometheus:v2.17.1
+```
+
+and then extract the installer from the prepared image:
+```
+$ oc adm release extract '--command-os=/usr/bin/openshift-install' --to=quay.io/vrutkovs/okd-release:4.4-updated-prometheus
+```
+This installer would now use this release as a source of truth - and use Prometheus v2.17.1 pulling it from Dockerhub.
+
+# Replacing other images
+
+Images included in the release payload are prepared on CI using Openshift builds from git repos (with one exception - `machine-os-content`, see below). In order to find out details about particular 
```

Comment:
would you mind adding a note about FCOS image too? if we are saying that is bundled part of the MCO then maybe we can add additional notes to avoid additional questions flowing down ?

Reply:
I was thinking I'll update `Building machine-os-content` section (in the follow up PRs) and reference it here?

Reply:
sounds good to me 👍

Reply:
find out details about **a** particular image, use